src/templates/best-cards/generic/animalism.html

```diff
@@ -1,115 +1,97 @@
+{% trans trimmed %}
+An excellent combat card. It is not a strike so it cannot be dodged,
+it is harder to prevent and additional damage are still inflicted by the strike itself.
+Moreover, it is ranged damage.
+{% endtrans %}
+{% endcall %}
+
+{% call layout.card_column("Cats' Guidance", decks=222, copies="2-5") %}
+{% trans trimmed %}
+A versatile defence tool providing either intercept or an unlock effect.
+Not as suprising as a wake, but being unlocked can be better.
+{% endtrans %}
+{% endcall %}
+
+{% call layout.card_column("Canine Horde", decks=195, copies="1-3") %}
+{% trans trimmed
+	carrion_crows=card("Carrion Crows"),
+	aid_from_bats=card("Aid from Bats")
+%}
+An excellent counter against weapons, it is often included in <i>i</i> combat modules.
+Because <i>i</i> often uses {{ carrion_crows }}
+and {{ aid_from_bats }} for a ranged combat module,
+a ranged weapon on the opponent side is especially cumbersome.
+{% endtrans %}
+{% endcall %}
+
+{% call layout.card_column("Deep Song", decks=194, copies="3-10") %}
+{% trans trimmed %}
+The choice between bleed and rush gives it so much versatility,
+it can be played in most table situations at an advantage.
+The fact that the opponent has to choose his strike first is a nice bonus.
+{% endtrans %}
+{% endcall %}
+
+{% call layout.card_column("Raven Spy", decks=182, copies="2-6") %}
+{% trans trimmed %}
+A base building block for most <i>i</i> based intercept modules.
+Permanent intercept is well worth the cost.
+{% endtrans %}
+{% endcall %}
+
+{% call layout.card_column("Sense the Savage Way", decks=167, copies="2-6") %}
+{% trans trimmed
+	eyes_argus=card("Eyes of Argus")
+%}
+Even if it is only available to big capacity vampires,
+this is intercept and wake built in the same card,
+a no-brainer if the deck has intercept options. It is even arguably better than <i>a</i>
+{{ eyes_argus }},
+as the intercept can be used on an undirected action.
+{% endtrans %}
+{% endcall %}
+
+{% call layout.card_column("Aid from Bats", decks=152, copies="6-14") %}
+{% trans trimmed
+	carrion_crows=card("Carrion Crows")
+%}
+The second pillar of <i>i</i> combat modules, it provides a maneuver, one damage, plus a welcome press,
+either as a defensive tool to avoid a second round at close range, or as an offensive tool to
+continue combat. Played multiple times in a row, it can take an opponent down
+pretty fast. Coupled with a {{ carrion_crows }},
+it delivers 6 damage with only two cards.
+{% endtrans %}
+{% endcall %}
+
+{% call layout.card_column("Army of Rats", decks=137, copies="1-2") %}
+{% trans trimmed
+	deep_song=card("Deep Song")
+%}
+<i>i</i> usually makes for excellent intercept and good combat,
+but it may lack pool damage to fence off one’s prey,
+with {{ deep_song }}
+bringing only a bit of pool damage.
+This is an excellent option to improve pool damage
+if the deck has enough intercept capabilities to protect it.
+{% endtrans %}
+{% endcall %}
+
```

Comment:
I would advise to add the matching translation in the same commit: that makes translations debugging easier than if you try to do it with many files at once - but you do it how you like it ☺︎